### PR TITLE
Make cart subtotal data type from `integer` to `float`

### PR DESCRIPTION
**Pull request by Jayendra88**
#### What does this PR do?
When we create a quote with large number of items which has subtotal greater than max value of `Int 32` will return an error response to the user. 
Since graphql schema allows `Float` value for cart `total`, I think we should allow `subtotal` also be `Float`.

#### How to test it? 
you can test this functionality in this [workspace](https://clouda--ecowaterb2b.myvtex.com/orderquote)

### CODE REVIEW

**Comment by Jayendra88** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by Jayendra88** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Review by wender — CHANGES_REQUESTED** [review]
It would be good to update the Schema version too otherwise it will not update to other customers

**Comment by Jayendra88** [comment]
Done

**Comment by Jayendra88** [comment]
> It would be good to update the Schema version too otherwise it will not update to other customers

Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.4] - 2021-01-12
```
Comment (wender) [inline CHANGELOG.md:10]: Remove the version, CI will do it automatically if it's under [Unreleased]

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "orderquote",
-  "version": "1.2.3",
+  "version": "1.2.4",
```
Comment (wender) [inline manifest.json:4]: Keep the same version as the original, CI will update the version automatically

**Review by wender — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Jayendra88) [reply CHANGELOG.md]: fixed

File: manifest.json (hunk 2 above)
Comment (Jayendra88) [reply manifest.json]: fixed

**Review by wender — APPROVED** [review]

**Pull request merged**